library/std/src/os/unix/net/ancillary.rs

```diff
@@ -418,6 +451,7 @@ impl<'a> SocketAncillary<'a> {
     /// let mut ancillary_buffer = [0; 128];
     /// let mut ancillary = SocketAncillary::new(&mut ancillary_buffer[..]);
     /// ```
+    // FIXME: add size example
```

Comment:
This is meant for a future PR or is this PR incomplete?

Reply:
It's incomplete.